lib/instance.js

```diff
@@ -236,15 +236,14 @@ module.exports = (function() {
 
         if (options.attributes) {
           keys = options.attributes;
-          if (this.Model._hasVirtualAttributes) {
-            keys = keys.concat(this.Model._virtualAttributes);
-          }
 
           if (this.options.includeNames) {
```

Comment:
How exactly are they being overwritten? `concat` concats.
It's not handled the same as before, before both code paths could be taken, now only one can.